
float32 x
float32 y
float32 y_Min
float32 y_Max
float32 idx
float32 dist 
